formats/utils/color-factory/utils.js

```diff
@@ -0,0 +1,51 @@
+const hueRotationFn = (rotation) => {
+  return (hue) => (360 + hue + rotation) % 360;
+};
+
+const saturationAdjustmentFn = (adjustment) => {
+  return (saturation) => Math.min(Math.max(saturation + adjustment, 0), 100);
+};
+
+function tokensToJson(data) {
```

Comment:
💅 Is there a reason this function is declared like this rather than using the same `const` syntax as the other functions above?

Reply:
Not sure I didn’t copy these over initially, but I can make the above functions, too.

Reply:
Convention in this repo prefers a function expression, so we should use:
```suggestion
const tokensToJson = (data) => {
```